### PR TITLE
Address changes from Scholarly Communications

### PR DESCRIPTION
**Pull request by CanOfBees**
**JIRA Ticket**: [IR-357](https://jirautk.atlassian.net/browse/IR-357)

# What does this Pull Request do?

This updates links in the `/user`'s page.

# How should this be tested?

# Interested parties
@markpbaggett

### CODE REVIEW

File: templates/user-profile.tpl.php [hunk 1]
```diff
@@ -98,8 +98,8 @@
     <h2>Submit Your Work</h2>
       <div style="line-height: 2.0; font-size: larger;">
           <ul style="list-style: none;">
-              <li style="padding-bottom:10px;"><a href='http://trace.tennessee.edu/submit_research.html'>Submit Faculty and Graduate Student Research and Creative Work</a></li>
-              <li style="padding-bottom:10px;"><a href='/islandora/object/utk.ir%3Atd/manage/overview/ingest'>Submit Graduate Theses and Dissertations</a></li>
+              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/submit_research.html'>Submit Faculty and Graduate Student Research and Creative Work</a></li>
+              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/utk_grad/'>Submit Graduate Theses and Dissertations</a></li>
```
Comment (markpbaggett) [inline templates/user-profile.tpl.php]:
```suggestion
              <li style="padding-bottom:10px;"><a href='https://trace.tennessee.edu/utk-grad/'>Submit Graduate Theses and Dissertations</a></li>
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]
@CanOfBees this looks pretty good, but I think we have the wrong url.  See suggestion.

File: templates/user-profile.tpl.php (hunk 1 above)
Comment (CanOfBees) [reply templates/user-profile.tpl.php]: :facepalm: thank you

**Comment by CanOfBees** [comment]
@markpbaggett I corrected that error, thanks for reviewing.

File: templates/user-profile.tpl.php (hunk 1 above)
Comment (markpbaggett) [reply templates/user-profile.tpl.php]: Just FYI:  you don't have to make these changes locally; you can use the commit suggestion button.

**Review by markpbaggett — APPROVED** [review]

**Pull request merged**